### ISSUE TITLE
Stuck at waiting for application...

### ISSUE THREAD

**Issue by felixlix45** [Grigory-Rylov/android-methods-profiler#94]
Using the latest release, got stuck when start a new trace. Need some insight

- Android Studio is closed
- Activity Name filled or not filled (same result)

Macbook Pro M1

**Comment by Grigory-Rylov** [comment]
Thanks for reporting! Was it an app or a plugin? Is application debuggable (or release with rooted device) ?

**Comment by esafirm** [comment]
I also have a similar but not the same situation. In my case, opening the app manually works.

**Machine:** 2018 Intel MBP
**Android App:** Debuggable on a non-rooted phone
**YAMP:** Latest 23.01.17, standalone

**Comment by Grigory-Rylov** [comment]
Hello, are you using correct main activity name? YAMP just trying to start that activity with ADB command like 'adb shell am start -n yourpackagename/.activityname' . Please try to execute that command in terminal with your package and activity name - will activity start?

**Comment by esafirm** [comment]
> Hello, are you using correct main activity name? YAMP just trying to start that activity with ADB command like 'adb shell am start -n yourpackagename/.activityname' . Please try to execute that command in terminal with your package and activity name - will activity start?

You're right, I got the wrong `Activity` name. Checking the log it was a `SecurityExcepiton` because `exported=false`.
Maybe we can have a better error message for this? 

Now the issue is shifting, the application opened but I got `Cannot find process <package name>` message again

Now the log shows this error:
```
om.github.grishberg.tracerecorder.exceptions.AppTimeoutException: <package_name> not started
```

**Comment by Grigory-Rylov** [comment]
> You're right, I got the wrong Activity name. Checking the log it was a SecurityExcepiton because exported=false.
Maybe we can have a better error message for this?

 Good idea!

> Cannot find process <package name>

Is the android studio currently running ? Android Studio holds debug port and other applications can't see debug process. 
You can use YAMP as the plugin. Thus  YAMP will reuse AS's debug port. Or you can kill android studio and try again.
